README.md

```diff
@@ -1,70 +1,195 @@
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+* Deepen our understanding and implementation of the modularity of React
+* Use React-Hooks to connect and contain our components and containers
+* Build on React Application using router, propTypes and fetching
+* Implement Context API for the holding of global state management
+* Test all Component flows and Asynchronous JavaScript using cypress
+
+## Evolution of the Project
+
+The project started with a broad vision of connecting all gender-related issues to their often broken up and hard to find resources.  As we came together as a team we narrowed our focus to get the app to take flight by beginning with reaching a queer audience of Colorado. From there, both a back end team and a front end team built out a github project board through a github organization, to help hold ourselves to the vision of the project.
+
+We intend to continue to work on this project as a team to make it ready for public use well after our experience at Turing School of Software Design.  This is a passion project through and through.
+
+## Reflections
+
+This project brought new learning challenges as we stretched ourselves as a team to understand `Circleci`, `React Hooks` and `rebase`.  The joy in the struggle of learning and growing was that this team was a delight to work with and we all walked away with hands on practice in `Global State Management` and `CI`.  It was a gift to see a dream-app come together with folks that care about the mission of the app.
+
+## Future Iterations
+
+* Security protection implementation to filter out bad actors
+* Make `CORS` more restrictive
+* Admin and Authorization
+* To increase our audience from queer based resources to include more gender-based needs
+* To Expand the app to include all of the United States
+
+## Authors
+#### Front End
+<table>
+    <tr>
+        <td> Angie Battillo <a href="https://github.com/battan40">GH</td>
+        <td> Marceline Ball <a href="https://github.com/MarcelineBall">GH</td>
+        <td> Riley Martin <a href="https://github.com/RMartin0717">GH</td>
+    </tr>
+    </tr>
+        <td><img src="https://avatars.githubusercontent.com/u/58871312?v=4" alt="A. Battillo" width="125" height="auto" /></td>
+        <td><img src="https://avatars.githubusercontent.com/u/65052658?v=4" alt="M. Ball" width="125" height="auto" /></td>
+        <td><img src="https://avatars.githubusercontent.com/u/76501236?v=4" alt="R. Martin" width="125" height="auto" /></td>
+    </tr>
+</table>
+
+#### Back End
+<table>
+    <tr>
+        <td> Austin Andrade <a href="https://github.com/austinandrade">GH</td>
+    </tr>
+    </tr>
+        <td><img src="https://avatars.githubusercontent.com/u/74944329?v=4" alt="A. Adrade" width="125" height="auto" /></td>
+    </tr>
+</table>
+
```

Comment:
I think we may want to add in a section for Aiden, even if it doesn't have a photo for him. I think we're gonna want a link to his github